robottelo/ui/user.py

```diff
@@ -109,13 +115,11 @@ def delete(self, name, really=True):
             locators['users.delete'],
         )
 
-    def update(self, username, new_username=None,
-               email=None, password=None,
-               first_name=None, last_name=None, locale=None,
-               roles=None, new_roles=None, locations=None,
-               new_locations=None, organizations=None,
-               new_organizations=None, default_org=None,
-               default_loc=None, select=False):
+    def update(self, username, new_username=None, email=None, password=None,
+               first_name=None, last_name=None, locale=None, roles=None,
+               new_roles=None, locations=None, new_locations=None,
+               organizations=None, new_organizations=None, default_org=None,
+               default_loc=None, select=False, submit=True):
```

Comment:
I think we dont need 'submit' parameter here. Because Update has to do submit operation only.
If we are not submitting the changes or cancelling the changes by setting submit=False, then there is no point in having update function.